python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -831,7 +831,9 @@ def queryDimensions(self, dimensions: Iterable[Union[Dimension, str]], *,
                         where: Optional[str] = None,
                         expand: bool = True,
                         **kwds) -> Iterator[DataCoordinate]:
-        # Docstring inherited from Registry.queryDimensions.
+        # Docstring inherited from Registry.queryDimensions
+        if isinstance(dimensions, str):
```

Comment:
@TallJimbo since you can take an iterable of `str` maybe the better option here is to use `lsst.daf.butler.core.utils.iterable` on the argument?